.github/workflows/build.yml

```diff
@@ -63,6 +63,13 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y dbus-x11 gnome-keyring libsecret-1-dev pass
+      -
+        name: Install gopass
+        if: ${{ matrix.os == 'ubuntu-20.04' }}
```

Comment:
Not needed

Reply:
good point; this was here prior to switching to using `go install` to install `gopass`.